python/pyspark/pandas/base.py

```diff
@@ -768,7 +768,7 @@ def ndim(self) -> int:
         """
         return 1
 
-    def astype(self, dtype: Union[str, type, Dtype]) -> Union["Index", "Series"]:
+    def astype(self, dtype: Union[str, type, Dtype]) -> Union["Index", "Series", "DataFrame"]:
```

Comment:
How does this happen?

Reply:
I think this is not a good idea because this never returns `DataFrame`.
@pingsutw Could you revert this change?

Reply:
Sorry to update so late. I was in the army a couple days ago.